### ISSUE TITLE
Merchants - Page directed at becoming a merchant

### ISSUE THREAD

**Issue by perrywoodin** [dashpay/dash-website#51]
from babygiraffe:

We need a section for merchants. The problem is that I would call that section “Merchants” which is already taken. Maybe rename the current “Merchants” section “Shop with Dash”. On this page, I would appeal to the merchant with the following main messages with a paragraph under each:
- Why accept Dash? Low fees, no chargebacks, no fraud… just fast easy payments
- Dash is built on Bitcoin’s code, making integration fast and easy
- Support InstantSend payments and provide users a superior buying experience
- Tap into Dash’s growing base of users eager to spend
- Dash even offers integration support
- Reach out to our business development team for additional information (link to Daniel’s email)

**Comment by andyfreer** [comment]
agree, spoke about this yesterday with the integrations team, will update on slack
